src/StructuredLogViewer.Core/Timeline/Timeline.cs

```diff
@@ -1,31 +1,97 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Microsoft.Build.Logging.StructuredLogger;
+using static System.Reflection.Metadata.BlobBuilder;
 
 namespace StructuredLogViewer
 {
     public class Timeline
     {
-        private Build build;
+        // parse for:
+        // midl.exe will run on 8 out of 8 file(s) in 8 batches.  Startup phase took 46.0028ms.
+        // Task 'CodeStore.idl' took 2244ms.
+        // Cleanup phase took 68ms.
+        const string regexStartupPhase = @"^.*\.exe will run on (?'activeFiles'[0-9]*) out of (?'totalFiles'[0-9]*) file\(s\) in [0-9]* batches\.\s*Startup phase took (?'msTime'[0-9]*.[0-9]*)ms.$";

```

Comment:
Consider moving these regexes to Strings.cs (it has examples of other regexes already)

Reply:
Moved to CppAnalyzer.